integration/integration_test.go

```diff
@@ -36,7 +36,7 @@ func getPiperExecutable() string {
 
 	f := piperutils.Files{}
 	wd, _ := os.Getwd()
-	localPiper := path.Join(wd, "piper")
+	localPiper := path.Join(wd, "..", "piper")
```

Comment:
Oh is the issue whether you're in the integration dir or not? Maybe we should check for both?

Reply:
No, seems to be related to the file and not where I run the `go test` command.

Reply:
@OliverNocon do you have a case where this worked?